test/PowerShellEditorServices.Test.Host/PowerShellEditorServices.Test.Host.csproj

```diff
@@ -21,15 +21,10 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.1.0-rc.1" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
-    <PackageReference Include="Microsoft.PowerShell.5.ReferenceAssemblies" Version="1.1.0" />
-  </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
-    <PackageReference Include="xunit" Version="2.4.1-pre.build.4059" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="PowerShellStandard.Library" Version="5.1.0" PrivateAssets="All" />
```

Comment:
Not sure due to the build failure that happens in `2.0.0` (also without any changes). It might be better to use `dotnet test` with a test adapter instead because the `dotnet xunit` has been deprecated a few months ago due to the problems that it has. I did this change in the PowerShell repo PR [here](https://github.com/PowerShell/PowerShell/pull/7980), a similar change could be applied here as welll.

Reply:
This won't work with PSStandard. It will call the PSStandard default API and fail. It needs to target the relevant PowerShell implementation.

Ideally we would actually load the thing as a module in a PowerShell exe and test it out that way. But that would mean moving us off xunit.